requirements/testing.txt

```diff
@@ -2,8 +2,11 @@
 
 django-dynamic-fixture==2.0.0
 
-pytest<4,>=3.3.2
+# 3.6.1 and >3.2.5 is incompatible
+# with pytest-describe 0.11.0
+pytest==3.2.5
```

Comment:
`pytest-describe` is incompatible with the current pytest (`3.6.1`) had to downgrade to `3.2.5` :anguished:. We can port those tests later

Reply:
Let's make an issue to address this later in the project.

Reply:
I opened an issue here https://github.com/rtfd/readthedocs.org/issues/4270